ZERC: Add Sapling and Overwinter Settings

mawenpeng opened the pull request:
ZERC activated Sapling and Overwinter. Without these settings, when submitting new blocks, ZERC daemon will return errors like:

```
2019-03-29 13:39:12 ERROR: ContextualCheckTransaction: overwinter is active
2019-03-29 13:39:12 ERROR: ProcessNewBlock: AcceptBlock FAILED
2019-03-29 13:39:12 ERROR: ReadBlockFromDisk: OpenBlockFile failed for CBlockDiskPos(nFile=-1, nPos=0)

```

mawenpeng committed:
Add Sapling and Overwinter Settings

ZERC activated Sapling and Overwinter. Without these settings, when submitting new blocks, ZERC daemon will return errors like:
-----
2019-03-29 13:39:12 ERROR: ContextualCheckTransaction: overwinter is active
2019-03-29 13:39:12 ERROR: ProcessNewBlock: AcceptBlock FAILED
2019-03-29 13:39:12 ERROR: ReadBlockFromDisk: OpenBlockFile failed for CBlockDiskPos(nFile=-1, nPos=0)
-----

egyptianbman commented on coins/zerc.json:
```diff
@@ -7,6 +7,8 @@
         "personalization": "ZERO_PoW"
     },
+    "sapling": 501000,
+    "overwinter": 501000,
```
Can you remove `overwinter`? If sapling is active, so is overwinter. Also, if ZERC is now passed block 501000, can you set this to true? Thanks!

mawenpeng replied:
Changed accordingly. Please review. Thanks.

mawenpeng committed:
Set sapling to true for ZERC

remove overwinter as sapling is active, so is overwinter. 
ZERC passed block 501000, set this to true.

egyptianbman reviewed (approved):
Thanks!
